addon/ng2/commands/new.ts

```diff
@@ -1,19 +1,15 @@
-'use strict';
```

Comment:
I believe you should add the typings at the first line;
`/// <reference path="../../../typings/main.d.ts" />`
also, run `typings install` in `package.json` in postinstall script.
If not, just ignore this.